library/alloc/src/vec/mod.rs

```diff
@@ -1549,8 +1549,7 @@ impl<T, A: Allocator> Vec<T, A> {
     #[track_caller]
     pub fn remove(&mut self, index: usize) -> T {
         #[cold]
-        #[inline(never)]
-        #[track_caller]
```

Comment:
Killed `track_caller`?

Reply:
Hrm, right. I had noticed it but not thought much of it. Is it required to get the panic to disappear? We could `#[cfg_attr(not(feature = "panic_immediate_abort"), track_caller)]

Reply:
🤦 

I deleted it because of inconsistency with all the other panic helpers. I think the better move is to add it to all the ones in that module.